zpool.go

```diff
@@ -1055,6 +1064,41 @@ func (pool *Pool) VDevTree() (vdevs VDevTree, err error) {
 	return
 }
 
+//Scrub starts/stop/pause scrubbing operation on zpool.
+func (pool *Pool) Scrub(cmd PoolScrubCmd) (err error) {
+
+	state, err := pool.State()
+	if err != nil {
+		return
+	}
+
+	if state == PoolStateUnavail {
+		err = fmt.Errorf("cannot scan %s : pool is currently unavailable", pool.Properties[PoolPropName].Value)
+		return
+	}
+
+	var scanType C.pool_scan_func_t = C.POOL_SCAN_SCRUB
+	var scrubCmd C.pool_scrub_cmd_t = C.POOL_SCRUB_NORMAL
+
+	switch cmd {
+	case PoolScrubStop:
+		scanType = C.POOL_SCAN_NONE
+	case PoolScrubStart:
+		break
```

Comment:
by default Go's switch doesn't fall through. what are we trying to do here?

Reply:
Yes, we dont need the break. I dont want the switch to fall to default.